### ISSUE TITLE
Why does main in package.json point to src

### ISSUE THREAD

**Issue by doonetheprune** [yuche/vue-strap#196]
Hi

Is there a reason why the main in the package.json points to `src/index.js` and not `dist/vue-strap.js` like it did in v1.0.8?

**Comment by simplesmiler** [comment]
[Blame the line](https://github.com/yuche/vue-strap/blame/master/package.json#L5) and figure out [where did the change come from](https://github.com/yuche/vue-strap/pull/158).

**Comment by prog-rajkamal** [comment]
@simplesmiler thanks for pointing to the issue.

@doonetheprune 
I am the person who made the changes. Is there a problem with it?

**Comment by doonetheprune** [comment]
@prog-rajkamal  the change you made now means a project using vue-strap has to do a build of vue-strap, which to me seems wrong. After your change I need to have a loader for `vue` and `sass` plus others. `main.js` should really looked at the built version of vue-strap.

I was looking at the reason for your change and not sure your change was needed as before you could just do the following to require in the vue-strap components.

`var aside = require('vue-strap').aside`

or the ES6 way

`var {aside} = require('vue-strap');`

**Comment by prog-rajkamal** [comment]
@doonetheprune 
Before the changes, `var aside = require('vue-strap').aside` was not working. 

The reason I changed main to "src/index.js" was because from the [comment] (https://github.com/yuche/vue-strap/issues/13#issuecomment-163075376 ) 

I believed that changes in  `main` was a workaround. and reverted main to `src\main.js` to see if it works. Well it did not work. so I changed to `src\index.js` and it did work.

Truth be told, the [npm docs](https://docs.npmjs.com/files/package.json#main) are so fantastic, I am not sure which is the correct way.

Well main points to dist in `vue`, so I believe you are correct.

I suggest make the changes locally, test if the ` require('vue-strap').aside` syntax works correctly with vueify and vue-loader, then make a PR for the same. 

I use vueify so I am happy to test it locally for vueify.

**Comment by doonetheprune** [comment]
@prog-rajkamal  I agree the npm docs are a vague on what you should do. I was basing my opinion on what some larger libraries are doing like bootstrap or jquery.

I have created a pull request #200 which moves the main in package.json back to dist, would you be able to test and see if it works with your setup?

Thanks

**Comment by prog-rajkamal** [comment]
@doonetheprune I will test it. and report about it here. 

Though, I think it would be better if ask opinion of someone who actually knows this stuff inside out. Do you know any one like that or a gitter channel where we can ask for advice?

**Comment by prog-rajkamal** [comment]
@doonetheprune 
Here is what i get after testing.

when `main` points to `dist/vue-strap.js`

---
> npm : [BABEL] Note: The code generator has deoptimised the styling of "path/to/dist/vue-strap.js" as it exceeds the max of "100KB".
>At line:1 char:1
>+ npm run build
>+ ~~~~~~~~~~~~~~~~~~~~~~~~
 >   + CategoryInfo          : NotSpecified: ([BABEL] Note: T...max of "100KB".:String) [], >RemoteException
 >   + FullyQualifiedErrorId : NativeCommandError
 
>Error: Cannot find module '!!./../node_modules/css-loader/index.js!./../node_modules/sass->loader/index.js!./spinner.scss' from 'path/to/dist'
 >   at path\to\node_modules\resolve\lib\async.js:46:17
  // and more stack trace

---

when `main` points to `src/index.js` and `spinner` component is commented out, it works correctly.


From all these, I believe that core problem is in *incorrect* building of `dist` files. the reason why it works with `src/index.js` at my local is that vueify builds vue-strap from src on the fly which is not desirable.

**Comment by doonetheprune** [comment]
@prog-rajkamal  It could be an issue with the `vue-strap` build or maybe something with your config. Could you reopen your old issue or maybe create a new one.

I will try and help you fix it, although not sure how useful I can be

**Comment by prog-rajkamal** [comment]
I am quite sure that problem is not in my config because I have used vue-cli's browserify template for exactly this reason.
